### ISSUE TITLE
Computed Infos using previous step data

### ISSUE THREAD

**Issue by amandavanice** [ovh/venom#664]
When using a test case with more then one step, the Computer Infos end up containing data from the previous step and not the current one.  This results in not being able to output the current result in 'info'

```
name: Tests
testcases:
  - name: MultiStep
    steps:
      - type: exec
        name: '1'
        script: echo 'step1'
        info: '{{.result.systemout}}'
      - type: exec
        name: '2'
        script: echo 'step2'
        info: '{{.result.systemout}}'
      - type: exec
        name: '3'
        script: echo 'step3'
        info: '{{.result.systemout}}'
```

Output:
```
[trac] writing output/venom.log
• Tests (siteActions/startHP.yaml)
 	• MultiStep PASS
	  [info] step1
	  [info] step1
	  [info] step2
Writing file output/test_results_siteActions_startHP.xml
Writing html file output/test_results.html
final status: PASS
```

Computed Infos:
<img width="1257" alt="1" src="https://user-images.githubusercontent.com/48518793/236033460-09123fcc-050d-401b-936f-4fb66c03e618.png">
<img width="1252" alt="2" src="https://user-images.githubusercontent.com/48518793/236033462-c4fb0561-d562-4108-bd67-e7fcc82e00da.png">
<img width="1254" alt="3" src="https://user-images.githubusercontent.com/48518793/236033457-97284ffe-f7e6-44ff-bf01-92a69a6d02a5.png">

**Comment by fokion** [comment]
yeap had the same issue

**Comment by fokion** [comment]
This issue looks similar with #622

**Comment by yesnault** [comment]
yes, fixed on v1.2.0.beta-2 version.

**Comment by jvalecillos** [comment]
I can confirm the same issue happening since version 1.0.0 and even earlier. And it is the same as in #658
